### DISCUSSION TITLE
Rule parity with Deno (`no-slow-types`, `no-sloppy-imports`)

### DISCUSSION THREAD

**Discussion by pauliesnug** [typescript-eslint/typescript-eslint#10169] [Community Feedback]
Does `typescript-eslint`, or `eslint` as a whole, have parity with the built-in [Deno CLI lints](https://github.com/denoland/deno/tree/main/cli/tools/lint/rules)? There are only two, but they are both important when publishing into a Deno-based ecosystem, and I think that achieving parity with these rules, or even the expanded `deno_lint` ruleset, would be pretty cool. Maybe there could be a plugin that implements rules from other linting tools that don't already exist in tseslint?

I know that `no-slow-types` already has some parity in TSESlint (`explicit-function-return-type`) but it's not a full solution from what I can see.

**Comment by bradzacher** [comment]
As per our contributing guide for discussions we do not use discussions for support or Q&A. Please read through our contributing for alternative forums to ask questions.

https://typescript-eslint.io/contributing/discussions
